express_relay/programs/express_relay/src/utils.rs

```diff
@@ -30,26 +30,19 @@ pub fn transfer_lamports_cpi<'info>(
     amount: u64,
     system_program: AccountInfo<'info>,
 ) -> Result<()> {
-    let cpi_accounts = Transfer {
-        from: from.clone(),
-        to: to.clone(),
-    };
+    if amount > 0 {
```

Comment:
it seems that we are doing this check externally too. let's remove it from here if it's already checked